tests/SignalSlot/RemoveTranslationSlotTest.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\PlatformHttpCacheBundle\Tests\SignalSlot;
+
+use EzSystems\PlatformHttpCacheBundle\PurgeClient\PurgeClientInterface;
+use EzSystems\PlatformHttpCacheBundle\SignalSlot\RemoveTranslationSlot;
+use eZ\Publish\Core\SignalSlot\Signal\ContentService\RemoveTranslationSignal;
+use eZ\Publish\SPI\Persistence\Content\Location;
+use eZ\Publish\SPI\Persistence\Content\Location\Handler as LocationHandler;
+use eZ\Publish\SPI\Persistence\Handler as PersistenceHandler;
+
+/**
+ * Test RemoveTranslationSlot for HttpCache.
+ */
+class RemoveTranslationSlotTest extends AbstractContentSlotTest
+{
+    protected $locationId = 43;
+    protected $parentLocationId = 45;
+
+    /**
+     * @var \eZ\Publish\SPI\Persistence\Handler|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $persistenceHandlerMock;
+
+    /**
+     * @var \EzSystems\PlatformHttpCacheBundle\PurgeClient\PurgeClientInterface|\PHPUnit_Framework_MockObject_MockObject
+     */
+    protected $purgeClientMock;
+
+    private $signal;
+
+    public function setUp()
+    {
+        $this->persistenceHandlerMock = $this->getMock(PersistenceHandler::class);
+        $this->purgeClientMock = $this->getMock(PurgeClientInterface::class);
```

Comment:
I expected all classes inheriting from parent to require fixes, but this apparently is not the case, so fixed in 0075477.